packages/testing/src/execution_testing/cli/benchmark_parser.py

```diff
@@ -270,18 +268,17 @@ def categorize_patterns(
 def generate_config_json(
     config: dict[str, list[int]],
     pattern_sources: dict[str, Path],
-) -> str:
+) -> OpcodeCountsConfig:
```

Comment:
I think setting the `default_counts` via `OpcodeCountsConfig` would be a little cleaner...
```suggestion
    default_counts: list[int],
) -> OpcodeCountsConfig:
```